docs/business-apps/power-apps/get-started/create-your-first-custom-form.md

```diff
@@ -106,11 +110,11 @@ To hide the **Attachments** field if the project owner is Nestor Wilke, follow t
 
     ![Select visible field](../../../images/gs01-attachments-property-visible.png)
 
-1. In the formula bar, enter the following formula: `If(SharePointIntegration.Selected.Owner.DisplayName = "Nestor Wilke", false)`
+1. In the formula bar, enter the following formula: `If(SharePointIntegration.Selected.Owner.DisplayName = "Nestor Wilke", false, true)`
 
     ![View-only title](../../../images/gs01-attachments-conditional-visibility.png)
     
-    The **Attachments** data card is now hidden if the project owner is Nestor Wilke.
+    The **Attachments** data card is now hidden if the project owner is Nestor Wilke. Otherwise, it is visible.
```

Comment:
If the project owner is Nestor Wilke, the **Attachments** data card is now hidden; otherwise, it is visible. <ed. note> This is standard tech writing or dev syntax in an if/then statement. Always have the condition first, and then the action. For example, we would write, "To use this formula, select **Save.**" It's incorrectly stated throughout much documentation as "Select **Save** to use the formula."